### ISSUE TITLE
no-require-imports false positive

### ISSUE THREAD

**Issue by adidahiya** [palantir/tslint#816]
```ts
import TestUtils = React.addons.TestUtils;
```

should not report a failure with the `no-require-imports` rule.
